bot.py

```diff
@@ -99,6 +102,97 @@ async def get_last_match(ctx, name: str):
     os.remove("df_styled.png")
 
 
+@bot.command(name="add", help="Add the players to the list")
+async def add_summoner(ctx, *, message):
+    """Writes list of summoners to local
+    json file and sends the list to the bot"""
+
+    json_path = "data/data.json"
+
+    # accepting comma, space and comma plus space
+    players_list = [x.strip() for x in message.replace(" ", ",").split(",")]
+    players_list = list(filter(None, players_list))
+
+    file_data = ""
+
+    # initializing number of players to display
+    number_of_players = 0
+
+    server_id = str(ctx.guild.id)
+
+    # storing the json file into a variable
+    if os.path.getsize(json_path) > 0:
+        with open(json_path, "r") as file:
+            file_data = json.load(file)
+
+        # if server id exist in json, add number of players
+        if server_id in file_data:
+            number_of_players += len(file_data[server_id])
+
+    # make dictionary for newly coming in players
+    players_list_info = create_summoner_list(players_list, server_id)
```

Comment:
You should only call API for 'data not pulled' summoners. 
This is going to call API even after we already pulled info in previous command

Reply:
fixed only call API for not pulled summoners with new push.
I don't know why pylint gives 10/10 when I run it locally but gives error when pylint checks on pr. So I just pushed again after fixing them.